blocks/story-carousel-block/features/story-carousel/default.jsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { Carousel, Image, Icon, Heading, Paragraph, Stack } from "@wpmedia/arc-themes-components";
+import { useContent } from "fusion:content";
+import { extractImageFromStory } from "@wpmedia/resizer-image-block";
```

Comment:
Blocks should not consume other blocks. We should make this utility available within the `arc-themes-components` package